pangeo_forge_recipes/storage.py

```diff
@@ -106,6 +109,18 @@ def from_url(cls, url: str):
         assert len(root_paths) == 1
         return cls(fs, root_paths[0])
 
+    def get_fsspec_remote_protocol(self):
+        """fsspec implementation-specific remote protocal"""
+        fsspec_protocol = self.fs.protocol
+        if isinstance(fsspec_protocol, str):
+            return fsspec_protocol
+        elif isinstance(fsspec_protocol, tuple):
+            return fsspec_protocol[0]
+        elif isinstance(fsspec_protocol, list):
+            return fsspec_protocol[0]
+        else:
+            raise ValueError("could not resolve fsspec protocol from underlying filesystem")
```

Comment:
Maybe we could return what fsspec_protocol is in the ValueError if it isn't caught by any of the isinstance checks?

Reply:
good catch